torch/_inductor/triton_heuristics.py

```diff
@@ -498,7 +498,7 @@ def benchmark_all_configs(self, *args, **kwargs):
             log.debug("Benchmark all input configs for %s, get:", self.fn.__name__)
             for k, v in timings.items():
                 log.debug(
-                    "%s: %f, nreg %d, nspill %d, #shared-mem %d",
+                    "%s: %f, nreg %d, nspill %d, #shared-mem %s",
```

Comment:
Any reason for this change?

Reply:
at least on my machine, shared memory is sometimes None for some reason. %s will jsut convert input to string regardless of type